gef.py

```diff
@@ -2230,6 +2234,23 @@ def get_info_sections():
 
     return
 
+def get_saved_ip():
+    """Retrieves the saved ip using the 'info frame' command."""
+
+    lines = gdb.execute("info frame", to_string=True).splitlines()
+    if lines:
+        saved_regs = lines[-1].split()
+        saved_ip_str = saved_regs[-1]
+
+        try:
+            saved_ip = int(saved_ip_str, 16)
+            return saved_ip
+        except ValueError:
+            # ignore
+            pass
+
+    return 0
+
```

Comment:
extra line

Reply:
Good morning,
I'm not clear on what to do here. Are you wanting me to add the extra line before get_saved_ip() or delete the extra line after it?  

In looking at the gef.py, it seems number of blank lines is consistent with PEP8, which is 2 blank lines after a functions (outside of a class) and 1 blank line after methods (within a class).  Thus, I'm assuming you are wanting me to add a line before the get_saved_ip().  Is this correct?

Reply:
No this is a mistake on my part, I thought that this was nested more. This is fine as it is.